felix/dataplane/windows/flattener.go

```diff
@@ -0,0 +1,267 @@
+// Copyright (c) 2019 Tigera, Inc. All rights reserved.
+
+package windataplane
+
+import (
+	"fmt"
+	"strconv"
+	"strings"
+
+	//"github.com/willf/bitset"
+	"github.com/bits-and-blooms/bitset"
```

Comment:
Just curious: why do we need `github.com/bits-and-blooms/bitset` vs.  `github.com/bits-and-blooms/bitset`? What is the difference between them?
And in any case a nit: we should remove the commented one...

Reply:
I added the `willf/bitset`, but golang complained that it's replaces with `bits-and-blooms/bitset`.
Will remove the old one.

Reply:
Removed.